app/navigation/OtherOfficialsNavigator.scala

```diff
@@ -133,7 +133,12 @@ class OtherOfficialsNavigator @Inject()(implicit frontendAppConfig: FrontendAppC
     }
 
     case OtherOfficialsNinoPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialsNinoPage(index)) match {
-      case Some(_) => routes.DeadEndController.onPageLoad() // TODO summary page
+      case Some(_) => redirectToPlaybackPage(index)
```

Comment:
can't use redirectToPlaybackPage here - you should be redirected to the summary page in CheckMode, not to the playback page

Reply:
ignore the comment above - this is the right way to do it, but wrong place; the ticket is to work on PlaybackMode, so don't worry about the CheckMode changes yet